apps/jenkins/src/main/fabric8/jenkins-maven-settings-secret.yml

```diff
@@ -1,7 +1,7 @@
+  settings.xml: "PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4NCjxzZXR0aW5ncz4NCiAgIDwhLS0gc2V0cyB0aGUgbG9jYWwgbWF2ZW4gcmVwb3NpdG9yeSBvdXRzaWRlIG9mIHRoZSB+Ly5tMiBmb2xkZXIgZm9yIGVhc2llciBtb3VudGluZyBvZiBzZWNyZXRzIGFuZCByZXBvIC0tPg0KICAgPGxvY2FsUmVwb3NpdG9yeT4ke3VzZXIuaG9tZX0vLm12bnJlcG9zaXRvcnk8L2xvY2FsUmVwb3NpdG9yeT4NCiAgIDwhLS0gbGV0cyBkaXNhYmxlIHRoZSBkb3dubG9hZCBwcm9ncmVzcyBpbmRpY2F0b3IgdGhhdCBmaWxscyB1cCBsb2dzIC0tPg0KICAgPGludGVyYWN0aXZlTW9kZT5mYWxzZTwvaW50ZXJhY3RpdmVNb2RlPg0KICAgPHNlcnZlcnM+DQogICAgICA8c2VydmVyPg0KICAgICAgICAgPGlkPm9zcy1zb25hdHlwZS1zdGFnaW5nPC9pZD4NCiAgICAgICAgIDx1c2VybmFtZSAvPg0KICAgICAgICAgPHBhc3N3b3JkIC8+DQogICAgICA8L3NlcnZlcj4NCiAgIDwvc2VydmVycz4NCiAgIDxwcm9maWxlcz4NCiAgICAgIDxwcm9maWxlPg0KICAgICAgICAgPGlkPm5leHVzPC9pZD4NCiAgICAgICAgIDxwcm9wZXJ0aWVzPg0KICAgICAgICAgPC9wcm9wZXJ0aWVzPg0KICAgICAgICAgPHJlcG9zaXRvcmllcz4NCiAgICAgICAgICAgIDxyZXBvc2l0b3J5Pg0KICAgICAgICAgICAgICAgPGlkPmNlbnRyYWw8L2lkPg0KICAgICAgICAgICAgICAgPHVybD5odHRwczovL3JlcG8xLm1hdmVuLm9yZy9tYXZlbjI8L3VybD4NCiAgICAgICAgICAgICAgIDxyZWxlYXNlcz4NCiAgICAgICAgICAgICAgICAgIDxlbmFibGVkPnRydWU8L2VuYWJsZWQ+DQogICAgICAgICAgICAgICA8L3JlbGVhc2VzPg0KICAgICAgICAgICAgPC9yZXBvc2l0b3J5Pg0KICAgICAgICAgPC9yZXBvc2l0b3JpZXM+DQogICAgICAgICA8cGx1Z2luUmVwb3NpdG9yaWVzPg0KICAgICAgICAgICAgPHBsdWdpblJlcG9zaXRvcnk+DQogICAgICAgICAgICAgICA8aWQ+Y2VudHJhbDwvaWQ+DQogICAgICAgICAgICAgICA8dXJsPmh0dHBzOi8vcmVwbzEubWF2ZW4ub3JnL21hdmVuMjwvdXJsPg0KICAgICAgICAgICAgICAgPHJlbGVhc2VzPg0KICAgICAgICAgICAgICAgICAgPGVuYWJsZWQ+dHJ1ZTwvZW5hYmxlZD4NCiAgICAgICAgICAgICAgIDwvcmVsZWFzZXM+DQogICAgICAgICAgICA8L3BsdWdpblJlcG9zaXRvcnk+DQogICAgICAgICA8L3BsdWdpblJlcG9zaXRvcmllcz4NCiAgICAgIDwvcHJvZmlsZT4NCiAgICAgIDxwcm9maWxlPg0KICAgICAgICAgPGlkPnJlbGVhc2U8L2lkPg0KICAgICAgICAgPHByb3BlcnRpZXM+DQogICAgICAgICAgICA8Z3BnLmV4ZWN1dGFibGU+Z3BnPC9ncGcuZXhlY3V0YWJsZT4NCiAgICAgICAgICAgIDxncGcucGFzc3BocmFzZT5teXNlY3JldHBhc3NwaHJhc2U8L2dwZy5wYXNzcGhyYXNlPg0KICAgICAgICAgPC9wcm9wZXJ0aWVzPg0KICAgICAgPC9wcm9maWxlPg0KICAgPC9wcm9maWxlcz4NCiAgIDxhY3RpdmVQcm9maWxlcz4NCiAgICAgIDwhLS1tYWtlIHRoZSBwcm9maWxlIGFjdGl2ZSBhbGwgdGhlIHRpbWUgLS0+DQogICAgICA8YWN0aXZlUHJvZmlsZT5uZXh1czwvYWN0aXZlUHJvZmlsZT4NCiAgIDwvYWN0aXZlUHJvZmlsZXM+DQo8L3NldHRpbmdzPg=="
```

Comment:
this setting has empty `<properties>` tag in nexus profile. Please, could you remove that?